assets/src/stories-editor/blocks/amp-story-text/edit.js

```diff
@@ -67,6 +78,20 @@ class TextBlockEdit extends Component {
 		if ( checkBlockDimensions ) {
 			maybeUpdateBlockDimensions( this.props );
 		}
+
+		// If the state changed to editing, focus on the text.
+		if ( this.state.isEditing && ! prevState.isEditing ) {
+			const textInput = document.querySelector( '.is-selected .wp-block-amp-amp-story-text' );
+			if ( textInput ) {
+				// Create selection, collapse it in the end of the content.
+				const range = document.createRange();
+				range.selectNodeContents( textInput );
+				range.collapse( false );
+				const selection = window.getSelection();
+				selection.removeAllRanges();
+				selection.addRange( range );
```

Comment:
Couldn't find a better way how to focus after going to editable state. See also https://github.com/WordPress/gutenberg/issues/9740.

Reply:
Maybe we can ask Ella or generally in #core-editor to find out if there's a better way.

Reply:
👍 Will do.